js/index.js

```diff
@@ -1 +1,131 @@
 // Your code goes here
+
+// 1 //
+// Select Dom Element
+const header = document.querySelector('header');
+// Create Event Listener
+header.addEventListener('mouseover', event);
+// Event Listener Call Back
```

Comment:
There's no need for both line 7 and 9. .addEventListener is looking for an event in the form of a string and a callback function. I like where this is going stylistically, but it would look more like this:
```
header.addEventListener('mouseover', header_mouseover);
function header_mouseover(event){
    header.style.backgroundColor = 'orange';
}

```